Formula/passenger.rb

```diff
@@ -47,29 +62,34 @@ def install
     # Ensure that the Phusion Passenger commands can always find their library
     # files.
 
-    locations_ini = `/usr/bin/ruby ./bin/passenger-config --make-locations-ini --for-native-packaging-method=homebrew`
+    locations_ini = `./bin/passenger-config --make-locations-ini --for-native-packaging-method=homebrew`
     locations_ini.gsub!(/=#{Regexp.escape Dir.pwd}/, "=#{libexec}")
     (libexec/"src/ruby_supportlib/phusion_passenger/locations.ini").write(locations_ini)
 
-    ruby_libdir = `/usr/bin/ruby ./bin/passenger-config about ruby-libdir`.strip
+    ruby_libdir = `./bin/passenger-config about ruby-libdir`.strip
     ruby_libdir.gsub!(/^#{Regexp.escape Dir.pwd}/, libexec)
-    system "/usr/bin/ruby", "./dev/install_scripts_bootstrap_code.rb",
+    system "./dev/install_scripts_bootstrap_code.rb",
       "--ruby", ruby_libdir, *Dir[libexec/"bin/*"]
 
-    system("/usr/bin/ruby ./bin/passenger-config compile-nginx-engine")
+    system "./bin/passenger-config", "compile-nginx-engine"
     cp Dir["buildout/support-binaries/nginx*"], libexec/"buildout/support-binaries", :preserve => true
 
-    nginx_addon_dir = `/usr/bin/ruby ./bin/passenger-config about nginx-addon-dir`.strip
     nginx_addon_dir.gsub!(/^#{Regexp.escape Dir.pwd}/, libexec)
```

Comment:
Is it intentional that this variable is used twice with difference values?

Reply:
Yes, one is the build time value, the other is the runtime value.